node-zerox/src/utils.ts

```diff
@@ -144,11 +145,38 @@ export const convertFileToPdf = async ({
   const outputPath = path.join(tempDir, outputFilename);
 
   try {
-    const pdfBuffer = await convertAsync(inputBuffer, ".pdf", undefined);
+    const pdfBuffer = await convertAsync(inputBuffer, "pdf", undefined);
     await fs.writeFile(outputPath, pdfBuffer);
     return outputPath;
   } catch (err) {
-    console.error(`Error converting .${extension} to .pdf:`, err);
+    throw err;
+  }
+};
+
+// Convert each page from an html to a pdf
+export const convertHtmlToPdf = async ({
```

Comment:
Seems to be working fine for me without this. Curious if this is just a local dev issue.

Reply:
What is the specific error that you were getting?